common/src/main/java/io/pravega/common/tracing/RequestTracker.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common.tracing;
```

Comment:
I'm not sure why we are putting this `tracing` logic in `common`. It is correct that it is not making any specific reference to Pravega, but I'm wondering whether this is more suitable for `shared`.

Reply:
I also had this doubt. At the moment I placed the code in `common` given the relationship of this work with logging (`LoggerHelpers` is in `common`), but it can be changed to `shared` if you think that this is more appropriate location.